api/Document.json

```diff
@@ -10605,17 +10533,6 @@
                     "value_to_set": "true"
                   }
                 ]
-              },
-              {
-                "version_added": "32",
-                "version_removed": "63",
-                "flags": [
```

Comment:
I'm not so sure about removing only part of the flag data for an entry, can you leave this in until it can all be removed?